modules/test/playwright/helpers/SearchExperiencesApiHelper.ts

```diff
@@ -0,0 +1,159 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {getRandomInt} from '../utils/getRandomInt';
+import {ApiHelpers, DataApiHelpers} from './ApiHelpers';
+
+const DEFAULT_SXP_BLUEPRINT_CONFIGURATION = {
+	advancedConfiguration: {},
+	aggregationConfiguration: {},
+	generalConfiguration: {
+		clauseContributorsExcludes: [],
+		clauseContributorsIncludes: ['*'],
+		searchableAssetTypes: [],
+	},
+	highlightConfiguration: {},
+	parameterConfiguration: {},
+	queryConfiguration: {
+		applyIndexerClauses: true,
+	},
+	sortConfiguration: {},
+};
+
+const DEFAULT_SXP_ELEMENT_DEFINITION = {
+	category: 'custom',
+	configuration: {},
+	icon: 'custom-field',
+};
+
+interface createSXPBlueprintProps {
+	configuration?: Object;
+	defaultLanguageId?: string;
+	description?: string;
+	elementInstances?: Object[];
+	title?: string;
+}
+
+interface createSXPElementProps {
+	defaultLanguageId?: string;
+	description?: string;
+	elementDefinition?: Object;
+	title?: string;
+}
+
+export class SearchExperiencesApiHelper {
+	readonly apiHelpers: ApiHelpers;
+	readonly basePath: string;
+
+	constructor(apiHelpers: ApiHelpers) {
+		this.apiHelpers = apiHelpers;
+		this.basePath = 'search-experiences-rest/v1.0';
+	}
+
+	/**
+	 * It allows creating an SXPBlueprint.
+	 *
+	 * @param configuration Technical configurations that include sort,
+	 * highlight, aggregation, and query configurations
+	 * @param defaultLanguageId the language id to assign for title/description
+	 * @param description the description of the blueprint
+	 * @param elementInstances the elements to be included in the blueprint
+	 * @param title the title of the blueprint
+	 */
+
+	async createSXPBlueprint({
+		configuration = DEFAULT_SXP_BLUEPRINT_CONFIGURATION,
+		defaultLanguageId = 'en_US',
+		description = '',
+		elementInstances = [],
+		title = `Blueprint${getRandomInt()}`,
+	}: createSXPBlueprintProps = {}): Promise<{id: number}> {
```

Comment:
Oh `={}` was just to prevent an error with calling `createSXPBlueprintProps()` instead of `createSXPBlueprintProps({})`... I'll update the types directly!